status/status.go

```diff
@@ -59,6 +59,8 @@ var transitioningMap = map[string]string{
 	"Updating":                    "updating",
 	"Waiting":                     "waiting",
 	"InitialRolesPopulated":       "activating",
+	"ScalingActive":               "pending",
+	"AbleToScale":                 "pending",
```

Comment:
I don't find references for these statuses in this and the related Rancher PR, are they valid?

Reply:
There are generated.

Reply:
It is valid. 
```
// ScalingActive indicates that the HPA controller is able to scale if necessary:
// it's correctly configured, can fetch the desired metrics, and isn't disabled.
ScalingActive HorizontalPodAutoscalerConditionType = "ScalingActive"

// AbleToScale indicates a lack of transient issues which prevent scaling from occurring,
// such as being in a backoff window, or being unable to access/update the target scale.
AbleToScale HorizontalPodAutoscalerConditionType = "AbleToScale"
```

Reply:
This is for the UI display usage. If HPA rule is not ready, you can see the reason from UI.